simulation-construction-set/falling-brick/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+   id("us.ihmc.ihmc-build") version "0.20.2"
+}
+
+ihmc {
+   group = "us.ihmc"
+   version = "0.0"
+   vcsUrl = "https://github.com/ihmcrobotics/ihmc-open-robotics-software-tutorials"
+   openSource = true
+
+   configureDependencyResolution()
+   configurePublications()
+}
+
+mainDependencies {
+}
```

Comment:
You need `api("us.ihmc:simulation-construction-set:0.16.0")` here.
I guess I forgot to mention last time that when trying to see if a dependency is actually useful, you can remove it from here, but then you need to refresh Gradle. When "refreshing Gradle", what happens is that Gradle is go through these scripts, reading the dependencies, and adding them to each project. This process is not automatic.